go/enclave/db/storage.go

```diff
@@ -458,6 +458,14 @@ func (s *storageImpl) StoreAttestedKey(aggregator gethcommon.Address, key *ecdsa
 	return obscurorawdb.WriteAttestationKey(s.db, aggregator, key)
 }
 
+func (s *storageImpl) FetchBatchBySeqNo(seqNum uint64) (*core.Batch, error) {
```

Comment:
What would `storage.FetchBatchByHeight` do?
1. Will it return a list of batches and let the caller determine which is the canonical one?
2. Or will it perform that logic in the storage?

Reply:
something has to.